### DISCUSSION TITLE
Welcome to Icaros Discussions! ❤️

### DISCUSSION THREAD

**Discussion by Xanashi** [Xanashi/Icaros#28] [Announcements]
## Welcome to the new Icaros Forum!
Hi Everyone,

From today onwards, this will be the place to get online Icaros support, 
and discuss the latest releases, bugs, feature request and so on. 

Be sure to use the **Watch** button in the upper right corner, to get notifications,
when your posts are replied to, new releases comes out, or even all activity on the Icaros Github page.

Finally, Thank you all for keeping the Icaros alive and motivating me to continuously improving
and adding new features to Icaros! :heart:

**Comment by mdOmarfaruk151** [comment]
hey, I like your work. 
I have used this software for a long time.
but when my Windows 11 update this software not working. Need .NET Framework to 4.8 to run. 
I request you please update this software as .NET Framework to 4.8 as soon as possible.

**Reply by Xanashi** [reply]
Hi mdOmarfaruk151,

There shouldn't be any issues with running Icaros on Windows 11. 
I'm on the latest development cycle of Win11 (Canary), and Icaros is running normally. 
It should also work completely fine with .NET Framework 4.8 installed.
Do you get an error message?

**Reply by thankyouforicaros** [reply]
Hi Xanashi!

Hey, so this is the only way i could find to contact you with what i wanted to ask you.  First of all just want to say HUGE fan of icaros.  Thanks so much for its amazing speed, reliability.   I wanted to ask if you could add a feature i've wanted for years.  Is it possible to add a feature where you can show the video's total time length in the bottom right of the generated thumbnails?  Like if a video was 7 min and 3 seconds long, if you could make it show up in white text in the bottom right of the thumbnails as "7:03"?  i would rlllllly love it if this was added.  hope u see this msg.  even if you are uninterested in adding this i still just want to say thx for icaros again  u rock

**Reply by Xanashi** [reply]
Thank you so much for your very kind words and support! It really means a lot to me.
I'll definitely consider your feature request. Could you create a ticket in the [Issues section](https://github.com/Xanashi/Icaros/issues)?
That way it will be easier for me to remember the request. ;)

**Reply by thankyouforicaros** [reply]
Hey, I think after having just now looked a bit that I never put in the ticket or maybe I did and can't find it who knows.  Just lettin u know I still havent let go of this feature desire.  I'm sure ur busy as me and anyone else with stuf to do, i'm just tryin to remind ya about this particular feature in the hopes u will bless us with it.  There is actually another program that does this feature but compared to icaros its total garbage outside having this one feature (timestamp of vid length overtop thumbnails in explorer) that I wanted for so long.  That program that can do this is called Video Thumbnailer (but again aside from having this one feature working on it, its glitchy af, slow af and just not good).  again thx for all ur work on icaros, i use it every single day man

**Reply by Xanashi** [reply]
It's definitely on the todo list. I can't say exactly when I will be able to get to it, 
but you can find an active issue on the feature [here](https://github.com/Xanashi/Icaros/issues/151).

**Comment by Latrell567** [comment]
Hi! I wanted to use your program to get thumbnails for my 10 bit video from my Sony a7S, but I have ARM64 which it doesn't run on with my Galaxy Book4 Edge. Is there any support coming soon?

**Reply by Xanashi** [reply]
Unfortunately ARM64 support has been a bit more complicated than I had hoped. 
I'm still looking into it, but I'm not quite there yet.
